src/main/java/org/netcracker/educationcenter/elasticsearch/database/operations/ElasticsearchOperations.java

```diff
@@ -0,0 +1,113 @@
+package org.netcracker.educationcenter.elasticsearch.database.operations;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.elasticsearch.ElasticsearchException;
+import org.elasticsearch.action.delete.DeleteRequest;
+import org.elasticsearch.action.delete.DeleteResponse;
+import org.elasticsearch.action.get.GetRequest;
+import org.elasticsearch.action.get.GetResponse;
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.action.index.IndexResponse;
+import org.elasticsearch.action.update.UpdateRequest;
+import org.elasticsearch.action.update.UpdateResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.common.xcontent.XContentType;
+import org.netcracker.educationcenter.elasticsearch.connection.Connection;
+
+import java.io.IOException;
+
+/**
+ * Interface for Elasticsearch Database operations (Insert, Get, Delete, Update)
+ *
+ * @author Mikhail Savin
+ */
+public interface ElasticsearchOperations {
+
+    /**
+     * @return JSON object mapper
+     */
+    ObjectMapper getMapper();
+
+    /**
+     * @return current connection instance
+     */
+    Connection getConnection();
+
+    /**
+     * @return index of the sought object
+     */
+    String getIndex();
+
+    /**
+     * Inserts given object (model) into the ES Database
+     *
+     * @param object object to insert
+     * @param id id of the inserted object
+     */
+    default void insert(Object object, String id) throws ElasticsearchOperationsException {
+        try {
+            String jsonString = getMapper().writeValueAsString(object);
+            IndexRequest indexRequest = new IndexRequest(getIndex())
+                    .id(id).source(jsonString, XContentType.JSON);
+            IndexResponse indexResponse = getConnection().getRestHighLevelClient()
+                    .index(indexRequest, RequestOptions.DEFAULT);
+        } catch (IOException e) {
+            throw new ElasticsearchOperationsException("Insert error", e);
+        }
+    }
+
+    /**
+     * Gets model's JSON as a String by its id.
+     *
+     * @param id searched JSON id
+     * @return searched JSON as a String
+     */
+    default String getById(String id) throws ElasticsearchOperationsException {
+        GetRequest getRequest = new GetRequest(getIndex(), id);
+        GetResponse getResponse;
+        try {
+            getResponse = getConnection().getRestHighLevelClient()
+                    .get(getRequest, RequestOptions.DEFAULT);
+        } catch (ElasticsearchException | IOException e) {
+            throw new ElasticsearchOperationsException("Can't get by id using getById() method", e);
+        }
+        if (getResponse != null && getResponse.isExists()) {
+            return getResponse.getSourceAsString();
+        } else {
+            return "Document was not found";
```

Comment:
This is not what user is expected to see when resource is not found
return Optional or null, (Optional is better)